### PR TITLE
Move search bar to nav

### PR DESCRIPTION
**Pull request by haiyingweng**
## Overview

Just some styling changes so search bar is on the nav and added CourseGrab logo

## Screenshots (delete if not applicable)

<details>

  <summary>Screen Shot Name</summary>

  
Screen Shot 2021-05-07 at 1 57 22 PM

</details>

### CODE REVIEW

File: src/Bell.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+</defs>
+</svg>
```
Comment (danielVebman) [inline src/Bell.svg:9]: Indentation in this file?￼

**Review by danielVebman — APPROVED** [review]
SO CLEAN 🔔🔔🔔

**Review by mathewscullin — APPROVED** [review]
Nav bar lookin tight!! 🙌

File: src/Bell.svg (hunk 1 above)
Comment (haiyingweng) [reply src/Bell.svg:9]: i think there's no indentation with .svg file

**Pull request merged**